doc/rest-api.md

```diff
@@ -1177,6 +1179,111 @@ Return:
 * Operation: Sync
 * Return: empty response or standard error
 
+## /1.0/containers/\<name\>/metadata
```

Comment:
@albertodonato did you miss this comment?

I think it's the only thing that still needs fixing before I can merge this.